waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -280,9 +283,11 @@
       (log/error "request to K8s API server failed: " url options body response)
       (ss/throw+ response))))
 
-(defn- service-description->namespace
-  [{:strs [run-as-user]}]
-  run-as-user)
+(defn- get-service-user
```

Comment:
Please add a docstring to explain what this is meant to return.

Reply:
Added